parse/parser.go

```diff
@@ -38,6 +38,8 @@ var operatorMap = map[TokenType]ast.Operator{
 	GTE:      ast.GTE,
 	EQ:       ast.EQ,
 	NOT_EQ:   ast.NOT_EQ,
+	Land:	  ast.LAND,
```

Comment:
Should be LAND?

Reply:
I make precedence name LAND and LOR.
So If I make Land to LAND, it has same name with precedence.
Also, other token name has lowercase like Slash, Mod, IntType and so on.
I think it is not bad to make name 'Land'.